internal/backend/db/dbgorm/dbtime/dbtime.go

```diff
@@ -0,0 +1,13 @@
+package dbtime
+
+import "time"
+
+var now = func() time.Time { return time.Now() }
+
+func Now() time.Time { return now() }
+
+// Used for testing.
+func Freeze() {
```

Comment:
1. The comment is useful in an IDE but not in a code review, if the function name also reflected that it's only for testing it would be safer (so reviewers who see this being called know that it must be only in tests)
2. Nit: the comment should be "<Function name> is used for testing"

Reply:
done